### ISSUE TITLE
Rethinking “Batch” in BatchNorm

### ISSUE THREAD

**Issue by yujiariyasu** [yujiariyasu/paper_summary#13]
# URL
https://arxiv.org/pdf/2105.07576.pdf

# Overview
・batch normの注意点の要約とその対処法をまとめた

# Point
・現状のBNは移動平均を使用するのがデファクトスタンダードになっているが、これは適切な統計量を生成しない可能性がある。代わりにPreciseBNを上手く使うことを推奨している。
・BNはtrain/testで異なる動作をするが、推論中にミニバッチ統計を使用するか、トレーニング中に母集団統計を使用することが有益な場合を示している。例えばR-CNN’s headでは母集団統計を使うよりミニバッチの統計量を使うほうがいい結果が得られる。
・複数のデータセットを使用しているため、またはレイヤーの共有のために、BatchNormの入力が異なるドメインからのものである状況についての考察。正規化するバッチの選択が最適ではない場合、ドメインシフトが発生する可能性があることを示している。
・BatchNormがトレーニング中にミニバッチ統計を使用すると、トレーニングサンプル間で情報が微妙にリークする可能性がある。この動作を研究し、正規化バッチを注意深く選択することで、この影響を軽減できることを示した。
